pkg/syncthing/syncthing.go

```diff
@@ -89,12 +88,18 @@ type Syncthing struct {
 	RemotePort       int          `yaml:"-"`
 	LocalGUIPort     int          `yaml:"-"`
 	LocalPort        int          `yaml:"-"`
-	Source           string       `yaml:"-"`
 	Type             string       `yaml:"-"`
 	IgnoreDelete     bool         `yaml:"-"`
 	pid              int          `yaml:"-"`
 }
 
+//Folder represents a sync folder
+type Folder struct {
+	Name       string `yaml:"name"`
```

Comment:
what's the name of the folder?

Reply:
I am just using numbers in sequence. this is not exposed to the user